internal/nonodo/nonodo.go

```diff
@@ -332,7 +332,7 @@ func NewSupervisor(opts NonodoOpts) supervisor.SupervisorWorker {
 					Model:              modelInstance,
 					Provider:           opts.RpcUrl,
 					InputBoxAddress:    common.HexToAddress(opts.InputBoxAddress),
-					InputBoxBlock:      opts.InputBoxBlock,
+					InputBoxBlock:      0,
```

Comment:
@fabiooshiro @ZzzzHui always start from zero?

Reply:
This value is not needed. So I set it to a default value 0 (it can be any value as it's ignored). 
User will specify the start block for espresso. And consequently, the start block for L1 is a value in the header of that espresso block.